apps/editor.planx.uk/src/pages/FlowEditor/components/Flow/index.tsx

```diff
@@ -16,18 +16,20 @@ export enum FlowLayout {
   LEFT_RIGHT = "left-right",
 }
 
-const Flow = ({
-  breadcrumbs = [],
-  lockedFlow,
-  showTemplatedNodeStatus,
-}: any) => {
+const Flow = ({ lockedFlow, showTemplatedNodeStatus }: any) => {
+  const { url } = useCurrentRoute();
+  const flowPath = url.pathname.split("/")[2];
+  const [_flow, ...breadcrumbIds] = flowPath.split(",");
```

Comment:
This fixes the persistent `Cannot read properties of undefined (reading 'split')` error we see (especially in dev mode).

Here's the solution - 
 - Only render this when a flow is actually loaded
 - Pass responsibility down to the correct component
 - use `useCurrentRoute()` hook instead of `window` call

<img width="1152" height="356" alt="Image" src="https://github.com/user-attachments/assets/8c051cf1-eef9-4f74-bafd-c3bceff9e8ad" />